src/python/fuzzing/strategy.py

```diff
@@ -57,6 +57,16 @@
     VALUE_PROFILE_STRATEGY,
 ]
 
+# List of strategies to include in multi-armed bandit query for libfuzzer
+libfuzzer_query_strategy_list = [
```

Comment:
Why does this need to be in this file? I think you can include it in `src/appengine/handlers/cron/fuzz_strategy_selection.py` as that seems to be the only place where this is used.

Reply:
had these here just because we define the strategy named tuple here, but yes makes more sense to keep in fuzz_strategy_selection